plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -606,6 +626,7 @@ def run(self):
         try:
             if self.trust_password is not None:
                 self.client.authenticate(self.trust_password)
+            self.ignore_volatile_options = self.module.params.get('ignore_volatile_options', default=True)
```

Comment:
That will result in an error, since `dict.get()` does not have a keyword argument.
```suggestion
            self.ignore_volatile_options = self.module.params.get('ignore_volatile_options', True)
```

Reply:
In any case, this will **not** do what you hope for. The key `ignorevolatile_options` is **always** present in `self.module.params`. Please simply keep
```suggestion
            self.ignore_volatile_options = self.module.params['ignore_volatile_options']
```
here and undo the other changes, i.e. put back the default value.